src/parser/statement.js

```diff
@@ -771,8 +771,13 @@ pp.parseClassBody = function (node) {
 };
 
 pp.parseClassProperty = function (node) {
+  const noPluginMsg = "You can only use Class Properties when the 'classProperties' plugin is enabled.";
+  if (!node.typeAnnotation && !this.hasPlugin("classProperties")) {
```

Comment:
This code will not stay forever anyway, as as soon as class-properties are final, we can remove all this.